repo/packages/H/hue/0/marathon.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "/hue",
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "mesosphere/hue",
+            "privileged": true,
+            "network": "BRIDGE",
+            "portMappings": [
+                {
+                    "containerPort": 8000,
+                    "hostPort": {{hue.port}},
+                    "protocol": "tcp"
+                },
+                {
+                    "containerPort": 14000,
+                    "hostPort": {{httpfs.port}},
+                    "protocol": "tcp"
+                }
+            ]
+        },
+        "volumes": [
+            {
+                "containerPath": "{{dcos.home}}",
+                "hostPath": "{{dcos.home}}",
+                "mode": "RO"
+            },
+            {
+                "containerPath": "/etc/hadoop/conf/",
+                "hostPath": "{{hadoop.conf-dir}}",
+                "mode": "RO"
+            }
+        ]
+    },
+    "uris": [
+        "{{hue.configuration}}",
+        "{{httpfs.configuration}}"
+    ],
```

Comment:
hi @abhay-agarwal, for this cmd, what you should do is directly download the Hadoop package that you need instead of mounting directories in DCOS image as volumes. We want to avoid dependencies on DCOS Image if possible, and it should be possible here. If you have questions about this, feel free to discuss with @cmaloney or ask me.